src/Backoffice/Airline/App/Requests/UpsertAirlineRequest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Airline\App\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Lightit\Backoffice\Airline\Domain\DataTransferObject\AirlineDto;
+use Lightit\Rules\NameUniqueForAirlines;
+
+class UpsertAirlineRequest extends FormRequest
+{
+    public const NAME = 'name';
+
+    public const DESCRIPTION = 'description';
+
+    public function rules(): array
+    {
+        return [
+            self::NAME => [
+                'string',
+                'max:255',
+                new NameUniqueForAirlines(),
+                $this->isMethod('post') ? 'required' : 'sometimes',
+            ],
+            self::DESCRIPTION => [
+                'string',
+                'max:255',
+                $this->isMethod('post') ? 'required' : 'sometimes',
+            ],
```

Comment:
```suggestion
            self::NAME => [
                'string',
                'max:255',
                new NameUniqueForAirlines(),
                'required',
            ],
            self::DESCRIPTION => [
                'string',
                'max:255',
                'required',
            ],
```
Since the update request is `PUT` it means it expects to get the entire set of data. Therefore, all fields are required.